x-pack/plugins/screenshotting/server/browsers/chromium/paths.ts

```diff
@@ -14,7 +14,7 @@ export interface PackageInfo {
   archiveChecksum: string;
   binaryChecksum: string;
   binaryRelativePath: string;
-  revision: 901912 | 901913;
+  revision: 901912 | 901913 | 1022525;
```

Comment:
I guess we can just have 1022525 now?

Reply:
https://github.com/vadimkibana/kibana/pull/1

Reply:
No, all 3 are still being used.